docs/overview.md

```
@@ -0,0 +1,181 @@
+- **Node.js**: 服务端应用集成
+- **Desktop**: Electron 桌面应用
+
+### 开发工具
+- Docker 开发环境
+- Git 子模块管理
+- Protobuf 协议定义
+- Mock 设备模拟器
+
+## 快速开始
+
+### 安装依赖
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)